include/dt-bindings/interrupt-controller/intel-ioapic.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef __DT_BINDING_IRQ_H
+#define __DT_BINDING_IRQ_H
+
+#define	LEVEL		0x00008000
+#define	EDGE		0x00000000
+#define	LOW		0x00002000
+#define	HIGH		0x00000000
+#define EDGE_RISING	0x00
+#define EDGE_FALLING	0x00002000
+#define LEVEL_HIGH	0x00008000
+#define LEVEL_LOW	0x0000A000
```

Comment:
Can EDGE_RISING, EDGE_FALLING, LEVEL_HIGH, LEVEL_LOW be defined like:

#define EDGE_RISING (EDGE | HIGH)
#define EDGE_FALLING (EDGE | LOW)